perf/istio-install/setup_prometheus.sh

```diff
@@ -25,9 +25,11 @@ function install_prometheus() {
   until timeout 60s kubectl get crds/podmonitors.monitoring.coreos.com; do echo "Waiting for CRDs to be created..."; done
   until timeout 60s kubectl get crds/prometheusrules.monitoring.coreos.com; do echo "Waiting for CRDs to be created..."; done
   until timeout 60s kubectl get crds/servicemonitors.monitoring.coreos.com; do echo "Waiting for CRDs to be created..."; done
-  helm template --namespace istio-prometheus "${INSTALLER_DIR}"/istio-telemetry/prometheus-operator/ -f "${INSTALLER_DIR}"/global.yaml --set prometheus.createPrometheusResource=true | kubectl apply -n istio-prometheus -f -
 
-  kubectl delete pod -l app=grafana
+  helm template --namespace istio-prometheus "${INSTALLER_DIR}"/istio-telemetry/prometheus-operator/ -f "${INSTALLER_DIR}"/global.yaml | kubectl apply -n istio-prometheus -f -
+
+  # Install Promethues
+  kubectl apply -f "${WD}/../prometheus-install/prometheus.yaml"
 }
```

Comment:
We do not need to specify the namespace as `istio-prometheus` here?

Reply:
All of the yamls have an explicit `namespace` set for them, so we do not need to set it here. Although maybe we should for consistency